frameworks/cassandra/tests/config.py

```diff
@@ -89,6 +93,15 @@ def run_backup_and_restore(
     delete_data_job = get_delete_data_job(node_address=job_node_address)
     verify_deletion_job = get_verify_deletion_job(node_address=job_node_address)
 
+
+    # Ensure the keyspaces we will use aren't present.
+    try:
+        jobs.run_job(delete_data_job)
+    except:
```

Comment:
I have no idea of the total set of exceptions that can come out here.  The correct design would be to write a job that doesn't fail if the data isn't there which would remove the need for the try block, but i don't see a way to do that in cqlsh.

Presumably if we're tossing an exception for reasons unrelated to the delete action, all the other jobs are going to be failing too.

I could probably spend a while trying to investigate the possible exceptions, but I suspect I'll be incomplete and this will introduce unintended flakiness.